assets/src/apps/authoring/components/RightMenu/PartPropertyEditor.tsx

```diff
@@ -335,7 +335,9 @@ export const PartPropertyEditor: React.FC<Props> = ({
   const selectPartType = selectedPartDef?.type || '';
 
   return (
-    <div className={`component-tab p-3 overflow-hidden part-property-editor ${selectPartType}`}>
+    <div
+      className={`component-tab p-3 overflow-hidden part-property-editor ${selectPartType}-part-property`}
```

Comment:
This change is coming from the https://github.com/Simon-Initiative/oli-torus/pull/5235. Instead of just applying the `part_type `as the class name, I just added 'part_type-part-property' so that the CSS that are meant for only the part component does not get apply to this part property fields.